src/Microsoft.ServiceHub.Framework/Extensions/ServiceHubUtilities.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.ServiceHub.Framework.Extensions
```

Comment:
nit: Can you switch to namespace statements instead of blocks?

Reply:
Done.